Detectors/TPC/workflow/src/CalibdEdxSpec.cxx

```diff
@@ -131,7 +133,7 @@ DataProcessorSpec getCalibdEdxSpec()
                                                                 false,                          // GRPECS=true
                                                                 false,                          // GRPLHCIF
                                                                 true,                           // GRPMagField
-                                                                false,                          // askMatLUT
```

Comment:
not sure about this one. Shouldn't askMatLUT only be activated if `material-type == 2` ?

Reply:
Yes, I think you are right, thanks. I will change this.